lib/Reader/readerir.cpp

```diff
@@ -1678,6 +1762,44 @@ Type *GenIR::getClassTypeWorker(
     }
   }
 
+  uint32_t EEClassSize = 0;
+  bool HaveClassSize = false;
+  if (!IsRefClass) {
+    try {
+      EEClassSize = getClassSize(ClassHandle);
+      HaveClassSize = true;
+    } catch (...) {
```

Comment:
It's HRException with hr set to E_FAIL. We currently don't have access to the coreclr eh.h file that declares that exception. Since the exception is so generic I didn't feel it was much better than ...